src/main/java/net/sf/jabref/model/entry/SpecialFields.java

```diff
@@ -0,0 +1,43 @@
+package net.sf.jabref.model.entry;
+
+
+public class SpecialFields {
+
+    public static final String FIELDNAME_PRINTED = "printed";
+    public static final String FIELDNAME_PRIORITY = "priority";
+    public static final String FIELDNAME_QUALITY = "qualityassured";
+    public static final String FIELDNAME_RANKING = "ranking";
+    public static final String FIELDNAME_READ = "readstatus";
+    public static final String FIELDNAME_RELEVANCE = "relevance";
+
+    public static final String PREF_SPECIALFIELDSENABLED = "specialFieldsEnabled";
```

Comment:
If these are values for preference keys, then these should be stored inside the JabRefPreference class